### DISCUSSION TITLE
(newbie) How can i get rid of these outlines/borders on floating windows?

### DISCUSSION THREAD

**Discussion by MxEggSalad** [i3/i3#6214] [Q&A]
whenever an application makes a floating window(?) it gets this gaudy outline on it. how can i fix this? 
![image](https://github.com/user-attachments/assets/f3b3132a-551f-4178-b23c-6d6cba31e5e3)
im using basically all of the default configurations that come with the Fedora 40 I3 swing, other than an installation of basic nvidia drivers

**Comment by mksonkar** [comment]
![image](https://github.com/user-attachments/assets/f3f5477e-543c-4e9e-a5ae-bac8efa8c55a)

I was just about to ask this question, and coincidentally I saw the latest question was regarding this.

What's causing the weird border and padding around the context menus?

**Comment by mksonkar** [comment]
The issue is because of picom shadows.
In the ~/.config/picom.conf

add the following to wintypes section:
```
wintypes:
{
  menu          = { shadow = false; };
  dropdown_menu = { shadow = false; };
  popup_menu    = { shadow = false; };
  utility       = { shadow = false; };
  toolbar       = { shadow = false; };
  tooltip       = { shadow = false; };
  splash        = { shadow = false; };
  dialog        = { shadow = false; };
  notification  = { shadow = false; };
  unknown       = { shadow = false; };
  combo         = { shadow = false; };
  dnd           = { shadow = false; };
}
```
additionally, the firefox class seems to have been changed from `Firefox` to `firefox`, so update it
```
  shadow-exclude = [
  ...
  "class_g = 'firefox' && argb",
  ...
]
```

**Reply by MxEggSalad** [reply]
ahhhh thank you so much this fixed it for me :D

**Reply by MxEggSalad** [reply]
hah wait no it didnt for some reason
